plugin_runner/plugin_runner.py

```diff
@@ -61,12 +68,29 @@ async def HandleEvent(self, request: Event, context):
 
             try:
                 protocol = protocol_class(request, plugin.get("secrets", {}))
+                compute_start_time = time.time()
                 effects = await asyncio.get_running_loop().run_in_executor(None, protocol.compute)
+                compute_duration = get_duration_ms(compute_start_time)
+                log.info(f"{plugin_name}.compute() completed ({compute_duration} ms)")
+                self.statsd_client.timing(
+                    f"plugins.protocol_duration_ms,plugin={plugin_name}",
+                    delta=compute_duration,
+                )
```

Comment:
We'll want to replace the `plugin_name` here to include the name of the protocol class. Plugins can include several protocols, we would want to see the timing differences between multiple protocols within a given plugin.

Reply:
I tested and it's showing both protocols:

```
INFO 2024-06-20 23:41:47,477 In the plugin
INFO 2024-06-20 23:41:50,478 joes_plugin:joes_plugin.protocols.my_protocol:Protocol.compute() completed (3001 ms)
INFO 2024-06-20 23:41:50,479 In the 2nd plugin
INFO 2024-06-20 23:41:51,480 joes_plugin:joes_plugin.protocols.my_2nd_protocol:Protocol.compute() completed (1000 ms)
INFO 2024-06-20 23:41:51,481 Responded to Event ASSESS_COMMAND__CONDITION_SELECTED (4003 ms)
```